stratagem-runner/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "stratagem-runner"
+version = "0.1.0"
+authors = ["IML Team <iml@whamcloud.com>"]
+edition = "2018"
+
+[dependencies]
+liblustreapi = { path = "../liblustreapi", version = "0.1.0" }
+csv = "1"
+serde = "1"
+serde_derive = "1"
```

Comment:
Don't need to add `serde_derive` separately, you can enable the derive feature on the serde crate:

`serde = { version = "1", features = ["derive"] }`

Reply:
I'll fix (I pulled this from the [csv documentation](https://docs.rs/csv/1.0.2/csv/))